### PR TITLE
build: upgrade azure-cloud-node-manager versions

### PR DESCRIPTION
**Pull request by lzhecheng**
Signed-off-by: Zhecheng Li <zhechengli@microsoft.com>

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -239,12 +239,16 @@
...
         "v1.0.6",
         "v1.0.8",
+        "v1.0.9",
         "v1.1.2",
         "v1.1.4",
-        "v1.23.1"
+        "v1.1.5",
+        "v1.23.1",
+        "v1.23.2"
       ]
```
Comment (luncj) [inline vhdbuilder/packer/components.json:252]: could we remove `v0.7.9` / `v1.0.6` / `v1.1.2`? @feiskyer

Comment (feiskyer) [reply vhdbuilder/packer/components.json:252]: yes, please remove the oldest patch version

Comment (lzhecheng) [reply vhdbuilder/packer/components.json:249]: done

**Comment by lzhecheng** [comment]
@feiskyer shall we merge this?

**Review by feiskyer — APPROVED** [review]

**Pull request merged**